docker/ci-py-go-themis.dockerfile

```diff
@@ -0,0 +1,91 @@
+FROM debian:buster
+
+SHELL ["/bin/bash", "-c"]
+
+# Install dependencies
+RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get -y install \
+    apt-transport-https \
+    build-essential \
+    ca-certificates \
+    curl \
+    default-mysql-client \
+    git \
+    gnupg \
+    libpq-dev \
+    libssl-dev \
+    openssl \
+    postgresql-client \
+    psmisc \
+    python3 python3-setuptools python3-pip \
+    rsync \
+    sudo \
+    wget
+
+WORKDIR /root
+
+# Install libthemis
+RUN set -o pipefail && \
+    curl -sSL https://pkgs.cossacklabs.com/scripts/libthemis_install.sh | \
+        bash -s -- --yes
+
+# Include helpful scripts
+RUN mkdir /image.scripts
+COPY docker/_scripts/acra-build/install_go.sh /image.scripts/
+RUN chmod +x /image.scripts/*.sh
+
+# Install Go
+RUN GO_PREFIX_DIR=/usr/lib/go/1.13.15 \
+    GO_VERSION=1.13.15 \
+    GO_TARBALL_DIGEST=01cc3ddf6273900eba3e2bf311238828b7168b822bb57a9ccab4d7aa2acd6028 \
+    GO_TARBALL_CLEAN=1 \
+    /image.scripts/install_go.sh
+
+RUN GO_PREFIX_DIR=/usr/lib/go/1.14.9 \
+    GO_VERSION=1.14.9 \
+    GO_TARBALL_DIGEST=f0d26ff572c72c9823ae752d3c81819a81a60c753201f51f89637482531c110a \
+    GO_TARBALL_CLEAN=1 \
+    /image.scripts/install_go.sh
+
+RUN GO_PREFIX_DIR=/usr/lib/go/1.15.2 \
+    GO_VERSION=1.15.2 \
+    GO_TARBALL_DIGEST=b49fda1ca29a1946d6bb2a5a6982cf07ccd2aba849289508ee0f9918f6bb4552 \
+    GO_TARBALL_CLEAN=1 \
+    /image.scripts/install_go.sh
+
+ENV GOROOT="/usr/lib/go/1.15.2/go" GOPATH="/home/user/gopath" GO111MODULE="auto"
```

Comment:
is it ok that we duplicate and hardcode this?

Reply:
This line (along with modified `PATH` few lines below) adds some default Go, so it's available out-of-the-box in the container. Only running `install_go.sh` won't affect env vars. We then may redefine `GOROOT` (and modify `PATH`) if some exact version is needed during tests. Not sure whether this is what you were asking about.